index.bs

```diff
@@ -3174,6 +3188,13 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
     1. Run [=obtain and deliver debug reports on source registration=] with
         « "<code>[=source debug data type/source-channel-capacity-limit=]</code>" » and |source|.
     1. Return.
+1. If |source|'s [=attribution source/attribution scope limit=] is not null:
+    1. Let |states| be the [=obtain a set of possible trigger states|number of possible trigger states=] obtained with |randomizedResponseConfig|.
```

Comment:
Can we compute |states| earlier and pass to the channel capacity algorithm to avoid  repeated computation?

Reply:
Done.